SDRF_template_droplet.txt

```diff
@@ -0,0 +1 @@
+Source Name	Comment[ENA_SAMPLE]	Comment[BioSD_SAMPLE]	Characteristics[organism]	Comment[cell count]	Comment[cell count pre filtering]	Comment[single cell quality] 	Material Type	Protocol REF	Extract Name	Comment[LIBRARY_LAYOUT]	Comment[LIBRARY_SELECTION]	Comment[LIBRARY_SOURCE]	Comment[LIBRARY_STRATEGY]	Comment[LIBRARY_STRAND]	Comment[NOMINAL_LENGTH]	Comment[NOMINAL_SDEV]	Comment[INSTRUMENT_MODEL]	Comment[single cell isolation]	Comment[library construction]	Comment[input molecule]	Comment[primer]	Comment[end bias]	Comment[spike in]	Comment[spike in dilution]	Comment[cdna read]	Comment[cdna read offset]	Comment[cdna read size]	Comment[cell barcode offset]	Comment[cell barcode read]	Comment[cell barcode size]	Comment[sample barcode offset]	Comment[sample barcode read]	Comment[sample barcode size]	Comment[umi barcode offset]	Comment[umi barcode read]	Comment[umi barcode size]	Protocol REF	Performer	Assay Name	Comment[technical replicate group]	Technology Type	Comment[ENA_EXPERIMENT]	Scan Name	Comment[ENA_RUN]	Comment[FASTQ_URI]	Comment[BAM_URI]	Comment[read1 file]	Comment[read2 file]	Comment[index1 file]	Comment[index2 file]
```

Comment:
would we need Comment[single cell quality] in droplet exps?

Reply:
we also might want to add Characteristics[inferred cell type] here as well - as we would like to include this in the SCEA (the SDRF becomes demultiplexed by cell)

Reply:
Yes, we are technically using the `Comment[single cell quality]` to filter out any runs that are labelled "not OK", even for 10x. I've seen one example where this was applicable (an experiment had some reads that had not been assigned to a sample). So leaving this in for now.

Reply:
Regarding the inferred cell type, this requires another solution in future. For now we cannot put this in the input SDRF.